packages/react-lowcode/src/codegen/index.ts

```diff
@@ -6,23 +6,13 @@ import ts, { factory } from "typescript"
 import { Project } from "ts-morph"
 import { HookImport } from '../ast/hooks'
 import { TagImport } from '../ast/tags'
-import { FacadeOptions, insertColumn, insertFormWidget } from './facade/facade-generator'
+import { insertColumn, insertFormWidget, deleteColumn as removeColumn } from './facade/facadeApi'
```

Comment:
we should unify `delete`, `remove` (maybe remove is proper one)

Reply:
Yes we should unify it. I agree with `remove`.